python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/traffic-policies.j2

```diff
@@ -7,34 +7,45 @@
 {% if traffic_policies is arista.avd.defined %}
 !
 traffic-policies
-{# Global options #}
-{%     if traffic_policies.options.counter_per_interface is arista.avd.defined(true) %}
-   counter interface per-interface ingress
-{%     endif %}
 {# FIELD SET #}
+{#     -- L4 PORT MANAGEMENT -- #}
+{%     if traffic_policies.field_sets.ports is arista.avd.defined %}
+{%         for field_set_port in traffic_policies.field_sets.ports %}
+   field-set l4-port {{ field_set_port.name }}
+{%             if field_set_port.port_range is arista.avd.defined %}
+      {{ field_set_port.port_range }}
+{%             endif %}
+   !
+{%         endfor %}
+{%     endif %}
 {#     -- IPv4 PREFIX MANAGEMENT -- #}
 {%     if traffic_policies.field_sets.ipv4 is arista.avd.defined %}
 {%         for field_set_ipv4 in traffic_policies.field_sets.ipv4 %}
+{%             if field_set_ipv4.name is arista.avd.defined %}
```

Comment:
Last time I missed to check sorting here.

Reply:
done